core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -460,6 +460,7 @@ public void render(float delta) {
      * also paused before it is destroyed.
      */
     public void pause() {
+        level.stopAllSounds();
```

Comment:
This doesn't seem to actually do anything, as I still hear the dog sounds when I'm paused.

Reply:
lol yes this exactly what I wrote